src/components/scene/stats.js

```diff
@@ -1,8 +1,8 @@
-var registerComponent = require('../../core/component').registerComponent;
-var RStats = require('../../../vendor/rStats');
-var utils = require('../../utils');
-require('../../../vendor/rStats.extras');
-require('../../lib/rStatsAframe');
+import { registerComponent } from '../../core/component.js';
+import RStats from '../../../vendor/rStats.js';
+import * as utils from '../../utils/index.js';
+import '../../../vendor/rStats.extras.js';
+import '../../lib/rStatsAframe.js';
```

Comment:
I think we can just remove the condition with module.exports in `src/lib/rStatsAframe.js`